apps/dashboard/src/utils/query-keys.ts

```diff
@@ -0,0 +1,11 @@
+interface IQueryKeys {
```

Comment:
The interface is not necessary. If we need the types we will get the `typeof` QueryKeys using TS inference.